### DISCUSSION TITLE
Github blocked account, support not responding

### DISCUSSION THREAD

**Discussion by mschoettner** [community/community#31584] [Other Feature Feedback, Questions, & Ideas]
For some reason, my account has been flagged. I contacted support about this, but haven't received any answer on my ticket for 22 days. I use this account for work and really need to get it unblocked. Do you know what I can do about this? It said that there is high traffic, but 22 days seems excessive to me.

**Comment by ghost** [comment]
you cant do anything. github is shit. move to gitlab.

**Comment by crmdevtool** [comment]
Hi, give me your nick and I'll see if support  answer.

**Reply by ghost** [reply]
shekhawat2
check this. i am waiting since 5days for a reply from github support. i have important stuffs on my account. i need it asap. after i make a backup, they can delete it again.
